Blueprints/BlueprintDefinitions/vs2022/NativeAOTServerless/template/src/BlueprintBaseName.1/Functions.cs

```diff
@@ -1,123 +1,96 @@
+using System.Text.Json.Serialization;
 using Amazon.Lambda.Core;
-using Amazon.Lambda.RuntimeSupport;
 using Amazon.Lambda.Serialization.SystemTextJson;
+using Amazon.Lambda.Annotations;
+using Amazon.Lambda.Annotations.APIGateway;
 using Amazon.Lambda.APIGatewayEvents;
-using System.Text.Json.Serialization;
-using System.Net;
+using BlueprintBaseName._1;
+
+[assembly:LambdaGlobalProperties(GenerateMain = true)]
+[assembly:LambdaSerializer(typeof(SourceGeneratorLambdaJsonSerializer<LambdaFunctionJsonSerializerContext>))]
 
 namespace BlueprintBaseName._1;
 
+/// <summary>
```

Comment:
This is the serverless Native AOT blueprint. It used to have its own `Main` method and logic for switching to the correct method based on an environment variable. Amazon.Lambda.Annotations can now do all of this for us so I removed all of the logic and relied on Lambda Annotations.